corehq/apps/users/forms.py

```diff
@@ -230,6 +231,11 @@ def direct_properties(self):
 
 
 class UpdateCommCareUserInfoForm(BaseUserInfoForm, UpdateUserRoleForm):
+    loadtest_factor = forms.IntegerField(
+        required=False, min_value=1, max_value=50000,
```

Comment:
yeah, i was thinking that maybe someone might want to make one case and then explode it to 50000. maybe that's too aggressive though.